src/lib/Theme.ts

```diff
@@ -818,7 +816,7 @@ export default class Theme {
         const currentContext = getActiveContext();
         switch (currentContext.theme_type) {
             case THEME_TYPE.react:
-                await Theme.syncReactTheme(currentContext);
+                await Theme.syncReactTheme(currentContext, undefined);
```

Comment:
why undefined is being passed instead of making it optional value

Reply:
fixed